docs/book/v3/cookbook/debug-toolbars.md

```diff
@@ -44,12 +44,78 @@ following contents:
 <?php
 
 use PhpMiddleware\PhpDebugBar\ConfigProvider;
+use Psr\Container\ContainerInterface;
 
-$provider = new ConfigProvider();
-return $provider();
+return array_merge(ConfigProvider::getConfig(), [
+    'dependencies' => [
+        'factories' => [
+            \PhpMiddleware\PhpDebugBar\PhpDebugBarMiddleware::class => \PhpMiddleware\PhpDebugBar\PhpDebugBarMiddlewareFactory::class,
+            \DebugBar\DataCollector\ConfigCollector::class => \PhpMiddleware\PhpDebugBar\ConfigCollectorFactory::class,
+            \PhpMiddleware\PhpDebugBar\ConfigProvider::class => function(ContainerInterface $container) {
+                return $container->get('config');
+            },
+            \DebugBar\DebugBar::class => \PhpMiddleware\PhpDebugBar\StandardDebugBarFactory::class,
+            \DebugBar\JavascriptRenderer::class => \PhpMiddleware\PhpDebugBar\JavascriptRendererFactory::class,
+        ]
+    ]
+]);
+```
+
+In addition, ensure these interfaces are registered as aliases in your container. For example, with Laminas Diactoros:
+
+```php
+return [
+    'dependencies' => [
+        'aliases' => [
```

Comment:
```suggestion
        'invokables' => [
```

Needs to be invokables for now, as:

- Diactoros does not register any services currently
- The container needs to know what to instantiate

Alternately, define aliases + factory configuration.